includes/Admin.php

```diff
@@ -142,6 +142,8 @@ public function enqueue_scripts() {
 				wp_enqueue_script( 'wc-enhanced-select' );
 			}
 		}
+
+		wp_enqueue_script( 'wc-facebook-google-product-category-fields', plugins_url( '/facebook-for-woocommerce/assets/js/admin/google-product-category-fields.min.js' ), [ 'jquery' ], \WC_Facebookcommerce::PLUGIN_VERSION );
```

Comment:
Can use `facebook_for_woocommerce()->get_plugin_url()` for this URL now that the plugin is frameworked

Reply:
Updated in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1542/commits/9b2148faf24012cb6ca1da4b845731db88dbfc26.